organizational-structure-overview.md

```diff
@@ -0,0 +1,87 @@
+# OpenSSF Governance & Org Structure
+
+## OpenSSF Charter
+
+The overall governance of the OpenSSF is defined in [the Foundation Charter](https://cdn.platform.linuxfoundation.org/agreements/openssf.pdf) (PDF file). 
+
+
+## Definitions
+
+The OpenSSF is comprised of instances of the following categories of official groups:
+
+- The **Governing Board** (GB) oversees the budget and legal status/structure of the OpenSSF. Board meetings may or may not be open to the public.
+- **Committees** are formed by, and report to, the Board. Broadly speaking, Committees handle non-technical matters as needed by the GB, wherein a named set of people are tasked with handling a specific objective. Like the Board meetings, Committee meetings may or may not be open to the public.
+- The **Technical Advisory Council** (TAC) is responsible for the general success of all Technical Initiatives (defined below). TAC meetings are generally open to the public, with the exception of special sessions. 
+  - TAC membership is limited to seven (7) seats, with alternating elections every year. Contributors to Technical Initiatves may run, and each contributor may vote once in TAC elections. 
```

Comment:
TAC membership is 7 seats, 4 of which are elected every year (not alternating), 3 of which are appointed by the Governing Board.

Reply:
Addressed in PR#117. I used the actual text from the charter.